### DISCUSSION TITLE
SSR prints so many logs

### DISCUSSION THREAD

**Discussion by shinrel22** [quasarframework/quasar#9931] [CLI - SSR mode] [answered]
Why It printed the log 6 times when I just entered this route for once? Is it quasar ssr bug or I'm doing it wrong?
![Screenshot from 2021-07-03 11-30-24](https://user-images.githubusercontent.com/21134858/124343042-c5adc380-dbf2-11eb-8a4c-95bdb4dbbe0e.png)

**Comment by rstoenescu** [comment, marked as answer]
100% not a Quasar SSR bug. Indeed you have 6 requests coming in. Browsers do lots of stuff behind the scenes to increase the perceived performance. Quick example: prefetching pages even while user is typing the URL in the addressbar... etc etc.
